uswds/uswds
# Update social icons

**Add LinkedIn icon.** We now have a LinkedIn icon included in our default icon sprite as `linkedin`.

**Use unmodified social media icons.** We now provide simpler social media icons, and removed any decoration not in the original icon. This means that the new icons are not in an enclosing circle unless the original icon is enclosed in a circle.

## Problem statement
Our social media icon set was missing the LinkedIn icon. Additionally, the social media icons we delivered were optimized for consistent presentation in our footer, inside a circular button. Since our footer is not the only place social media icons might be used, we should remove the circular outline around the images, since we do not want to imply that the circle is part of the icon itself.

## How to test
- Open the Icons component of our Storybook site and look for the `linkedin` icon
- In the Icons component of our Storybook site, check the following icons to assure that they do not have a circular button shape. Note that the Facebook and GitHub are circular and have not changed. 
  - Flickr
  - Instagram
  - Twitter
  - YouTube

## Examples
### New Instagram icon in our Icons component
<img width="988" alt="Screen Shot 2022-07-20 at 10 43 36 AM" src="https://user-images.githubusercontent.com/11464021/180049420-666a7c8e-5e4c-447a-9fb9-68fb011f2ad4.png">

### All social icons in the footer
Note that this a is just an example. This PR does not add social media icons to the footer.

<img width="520" alt="Screen Shot 2022-07-20 at 10 34 21 AM" src="https://user-images.githubusercontent.com/11464021/180049596-bba33ed9-fa5c-44da-99c4-88941ae4ecf9.png">
